README.md

```diff
@@ -1 +1,47 @@
-# Technical-documentation-page-new
\ No newline at end of file
+# Technical-documentation-page-new
+
+This is a documentation page about CSS (Cascading Style Sheets), covering various topics such as what is CSS, Anatomy of a CSS ruleset, Selecting Multiple elements, Diffrenet types of selectors, and CSS boxes. It serves as an educational resource for understanding and knowing of CSS.
+
+## Table of Contents
+
+- [Introduction](#Introduction)
+- [What is CSS](#whatisCSS)
+- [Anatomy of a CSS ruleset](#AnatomyofaCSSruleset)
+- [Selecting multiple elements](#Selectingmultipleelements)
+- [different types of selectors](#differenttypesofSelectors)
+- [CSS: all about boxes](#CSSallaboutboxes)
+- [References](#References)
+
+## Sections
+
+Each section of the documentation covers a different aspect of CSS and Different types, providing both descriptive text and relevant information.
+
+## References
+
+At the end of the documentation, there is a section providing links to additional resources for those who want to explore further.
+
+# Demo 
+![alt text](DemoPic.PNG)
+<video controls src="webpage.mp4" title="Title"></video>
+
```

Comment:
- There is no video in your project named `webpage.mp4`.
  - Your video should be short in size and length so you can add and push it to GitHub otherwise GitHub does not accept it.
  - Instead use the online resources for taking a PNG from all of your project's pages or add a screenshot of your project.